scripts/run_tests.sh

```diff
@@ -3,25 +3,33 @@
 # Run project tests
 #
 # NOTE: This script expects to be run from the project root with
-# ./scripts/run_tests.sh
-
-# Use default environment vars for localhost if not already set
-export EQ_CLOUDWATCH_LOGGING=False
+# ./scripts/run_tests.sh [-u] [-f]
```

Comment:
Looks like we are adding a lot of logic to support option parameters. Why not split the two bits of logic into `run_tests_functional.sh` and `run_tests_unit.sh` where this run_tests.sh wraps both?